### PR TITLE
fix belgium,dk,usa

### PR DESCRIPTION
**Pull request by lordfolken**

### CODE REVIEW

File: data/airspace.json [hunk 1]
```diff
@@ -227,10 +227,10 @@
     },
     {
       "name": "Spain_Airspace.txt",
-      "uri": "https://soaringweb.org/Airspace/ES/SUASpain202004.full.txt",
+      "uri": "https://soaringweb.org/Airspace/ES/SUASpain202104.full.txt",
       "type": "airspace",
       "area": "es",
-      "update": "2020-04-01"
+      "update": "2021-04-03"
```
Comment (rawtaz) [inline data/airspace.json:233]: In the file it says `Dated 3 May 2021`, even though the filename is 2021**04**. I guess it makes more sense to put `2021-05-03` as the date here in our reference.

Comment (lordfolken) [reply data/airspace.json]: Good catch.

**Comment by rawtaz** [comment]
Are the README.md changes intended to be in this PR?

**Comment by lordfolken** [comment]
> Are the README.md changes intended to be in this PR?

Yes I think it's ok. As long as the CI and deploy

**Review by rawtaz — APPROVED** [review]
/me applies [approved] stamp to forehead of @lordfolken

**Pull request merged**